### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 12

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#277] [Esercizi - Viaggi/Piazza]
**Esercizio 5.8 del libro di testo (Geometria Analitica: Marco Abate & Chiara De Fabritiis)**

**Comment by daromacarlo** [comment]
![Immagine 2023-12-04 175833](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/2b25b0f6-76a5-40e4-95ac-414021359626)

**Reply by 18xChris** [reply]
sbaglio oppure si poteva evitare di fare il sistema dato che il sistema è quadrato e quindi automaticamente ker = 0?

**Reply by Elia-Belli** [reply]
@Entscheidungs in effetti si, siccome per trovare il kernel devi risolvere un sistema omogeneo (che ha sempre come soluzione il vettore nullo) e la matrice triangolare superiore ha tutti i pivot non nulli (unica soluzione) allora il vettore nullo è soluzione unica.
